src/status.js

```diff
@@ -0,0 +1,110 @@
+import addEventsDragAndDrop from './sorting.js';
+
+let collection = [];
+const form = document.getElementById('addTodo');
+
+function stausCheck(ev) {
+  const buttonId = ev.target.id;
+  const dataGet = localStorage.getItem('todoObject');
+  const data = JSON.parse(dataGet);
```

Comment:
- [ ]  In your file 'staus.js'  inside each function, there are some local variables. And for each local variable, you have used 'const' instead of 'let' or 'var'.For detail you can use this link: [Always Declare Local Variables[](https://www.w3schools.com/js/js_best_practices.asp)

Reply:
I try to replace const with let and it gave linter errors that let variable value is never reassigned so use const instead of let.